### PR TITLE
Regenerate controller with codegen `v0.19.3`

### PR DESCRIPTION
**Pull request by A-Hilaly**
Issue: N/A

Description of changes:
- Bump runtime version to 0.19.3
- Regenerate controller with code-generator `v0.19.3`

By submitting this pull request, I confirm that my contribution
is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: generator.yaml [hunk 1]
```diff
@@ -17,15 +21,6 @@ resources:
...
-          - method: Create
-            ignore: true
-      LayerStatuses:
-        is_read_only: true
-        from:
-          operation: GetFunction
-          path: Configuration.Layers
     renames:
       operations:
         CreateFunction:
```
Comment (jaypipes) [inline generator.yaml:26]: these deletions (and the ones below) in the generator.yaml file indicate functional changes to the Lambda controller's CRDs and controller implementation that are unrelated to the v0.19.2 runtime upgrade... can you explain?

**Comment by A-Hilaly** [comment]
Tests should pass after #38 merge

**Comment by jaypipes** [comment]
@A-Hilaly this is failing in test runs:

```
 >       assert function["Tags"] == tags
E       AssertionError: assert {'services.k8...2': 'k2', ...} == {'v1': 'k1', ...', 'v3': 'k3'}
E         Omitting 3 identical items, use -vv to show
E         Left contains 2 more items:
E         {'services.k8s.aws/controller-version': 'lambda-v0.0.16-3-gbe97a1a',
E          'services.k8s.aws/namespace': 'default'}
E         Full diff:
E           {
E         +  'services.k8s.aws/controller-version': 'lambda-v0.0.16-3-gbe97a1a',...
E         
E         ...Full output truncated (6 lines hidden), use '-vv' to show
```

You need to use the [`tags.assert_equal_without_ack_tags()`](https://github.com/aws-controllers-k8s/test-infra/blob/58ffb067d7c400ffbd0d470e975056efdc9e4965/src/acktest/tags.py#L119) functionality to assert tags without regard for the ACK default tags...

**Comment by A-Hilaly** [comment]
@jaypipes tests are passing now.
